ODIN_II/dt.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
```

Comment:
this file should be under the tools directory in the odin-ii/regression_tests
Also, you need to run `make format-py` and `./dev/pylint_check.py` from the VTR_ROOT directory

The same for C/C++ code -> `make format`